### DISCUSSION TITLE
PineconeConnectionError: Request failed to reach Pinecone while calling https://controller.6328f49....

### DISCUSSION THREAD

**Discussion by stccorp** [FlowiseAI/Flowise#1068] [General]
Hello I am getting this  Marketplace Conversational AI QA.  One of the problems is that 100% of the samples  that you can find on the Internet are wrong since the screen for pinecone  changed in flowwise. Before it ask for an environment and an index.  Now only ask for index. There are 2 ways you could do this.  Use the name of the index or pinecone gives you an url for the index.  I am assuming that the url is what we should use since it has the env already as part of the index.   In any case, I get the error using the url of the index or the name.

HOST https://XXXXXXXXXX.svc.us-west1-gcp-free.pinecone.io
or test1

Any ideas?
Thank you

**Comment by PingUmich** [comment]
Did you fix the problem? I have the same issue. Could you give me some advice if you fixed the problem? Thank you
